code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -198,7 +198,7 @@
 
 /obj/item/explosive/grenade/incendiary
 	name = "\improper M40 HIDP incendiary grenade"
-	desc = "The M40 HIDP is a small, but deceptively strong incendiary grenade designed to disrupt enemy mobility with long-lasting Type B napalm. It is set to detonate in 4 seconds."
+	desc = "The M40 High Incendiary Dual-Purpose (HIDP) is a small, but deceptively strong incendiary grenade designed to disrupt enemy mobility with long-lasting Type B napalm. It is set to detonate in 4 seconds."
```

Comment:
```suggestion
	desc = "The M40 High Incendiary Dual-Purpose (HIDP) is a small but powerful incendiary grenade, designed to disrupt enemy mobility with long-lasting Type-B napalm. It is set to detonate in 4 seconds."
```

Reply:
renamed all due to lore reasons and applied these